src/core/Cosmetics.ts

```diff
@@ -0,0 +1,87 @@
+import { base64url } from "jose";
+import rawTerritoryPatterns from "../../resources/cosmetic/cosmetic.json" with { type: "json" };
+import { CosmeticsSchema } from "./CosmeticSchemas";
+
+export const territoryPatterns = CosmeticsSchema.parse(rawTerritoryPatterns);
+
+export class PatternDecoder {
+  private bytes: Uint8Array;
+  private tileWidth: number;
+  private tileHeight: number;
+  private scale: number;
+  private dataStart: number;
+
+  constructor(base64: string) {
+    const bytes = base64url.decode(base64);
```

Comment:
This code has also reverted back to an older version. Please ensure that all comments have been applied.

```suggestion
    this.bytes = base64url.decode(base64);
```

Reply:
It’s not really that it reverted — I think this is just a complete oversight. As far as I remember, this part never used this.bytes in the first place.